src/constants.h

```diff
@@ -12,12 +12,9 @@
 #define GLOBAL_SURF_IN_DIM_MAX 5005 // maximum dimension of global surface input files
 #define BASIN_SURF_IN_DIM_MAX 3005 // maximum dimension of basin surface input files
 #define MAX_FILENAME_STRING_LEN 1024 // maximum dimension of basin surface input files
-#define LON_GRID_DIM_MAX 10260 // X
-#define LAT_GRID_DIM_MAX 19010 // Y
+#define LON_GRID_DIM_MAX 30000 // X  - used to be 10260 in non-MPI version
```

Comment:
I think it is a good idea to have a conditional compilation with `#ifdef MPI` that is defined by `-D MPI` as a C compiler option through makefile.mpi.

Reply:
My vision is that we merge in Daniels changes sooner rather than later (before the merge conflicts set in). As AWP VM generation is something we'll need later.

I'm neither here nor there about the MPI. I do agree that in an ideal world the conditional compilation would be a good thing. However, we could put it in the pile of stuff to do and do it when it becomes an issue rather than being proactive about something that may not be used - especially has no intention of being used in the primary workflow.

Let's chat after standup about AWP and this change.